spec/models/gogovan_order_spec.rb

```diff
@@ -14,13 +14,12 @@
     it{ is_expected.to have_db_column(:driver_license).of_type(:string)}
   end
 
-  describe 'save_booking' do
+  describe 'update_booking' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.